### ISSUE TITLE
"Attempted" date in additional information is actually "Graded" date

### ISSUE THREAD

**Issue by anieminen** [gjb2048/moodle-format_topcoll#118]
Hi,

Quiz activity shows "Attempted" date to students under the activity name. We have noticed that it is actually the date when the student's responses were last graded. Quizzes may have essay questions which are graded afterwards, so the date can change. Is this a bug? 

Course front page as a student:
![quiz_attempted](https://user-images.githubusercontent.com/15781654/145828020-db2c1ece-c718-46f4-87ff-4e3db82986b0.PNG)

Quiz summary page:
![quiz_summary](https://user-images.githubusercontent.com/15781654/145828053-fbf216f8-216d-4047-8646-16c4b6cd1215.PNG)

**Comment by gjb2048** [comment]
Yes and no, but then there would be no way for the code to detect this if the quiz is reporting via the data in the database that its been graded.

**Comment by gjb2048** [comment]
It's a nightmare, so not going to do after all of this time.
